debian/control

```diff
@@ -15,11 +15,21 @@ Depends: clearwater-infrastructure, clearwater-nginx, libxml2-dev, libxslt1-dev,
 Suggests: clearwater-logging, clearwater-snmpd, clearwater-secure-connections
 Description: crest, a Cassandra powered generic RESTful HTTP server platform
 
+Package: homer-node
+Architecture: any
+Depends: homer
+Description: homer-node, the Cassandra powered XDMS
+
 Package: homer
 Architecture: any
 Depends: crest
 Description: homer, the Cassandra powered XDMS
 
+Package: homestead-node-prov
```

Comment:
So having a homestead-node-prov package doesn't really make much sense, as we can't really deploy just homestead-prov on a node on its own. 
Instead, i think we should just keep it as the single homestead-prov package, and have it as a dependency of the homestead-node package, as we always say that both should be installed at the same time anyway